great_expectations/core/expectation_suite.py

```diff
@@ -740,7 +740,7 @@ def add_expectation_configurations(
         return expectation_configurations_attempted_to_be_added
 
     @public_api
-    def add_expectation(
+    def add_expectation_configuration(
```

Comment:
Should this be public api?

Reply:
no, but that's a more significant refactor